python/equistore-operations/equistore/operations/_dispatch.py

```diff
@@ -330,7 +339,7 @@ def ones_like(array, shape=None, requires_grad=False):
         raise TypeError(UNKNOWN_ARRAY_TYPE)
 
 
-def empty_like(array, shape=None, requires_grad=False):
+def empty_like(array, shape: Optional[List[int]] = None, requires_grad: bool = False):
```

Comment:
I don't think this should be part of this PR

Reply:
`zeros_like` and `empty_like` of `_dispatch` are used inside the reduce operations. I'll make sure to rebase

Reply:
Rebased